README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+### Brief Biography - Nelson O. Puc Cruz
+##### HW1
+
+I was born and raised in Chetumal, a city of around 150,000 people in the southern state of Quintana Roo, México. The first child to my parents and grandchild to theirs, my family was ecstatic over my arrival (or so I've been told), although the only advantage that came out of my earliness was to work as a private driver for my two younger siblings on later years. I studied my first years of education in schools I don't really remember the name of, until I attended high school at the _Colegio de Bachilleres Plantel Chetumal 1_. This was followed by 4 and a half years of solving equation systems, which granted me a bachelor's degree in Economics and Finance, awarded by the _Universidad de Quintana Roo_. After graduating, I worked at the state government until the rampant corruption forced me to consider a change of career (no, really). I decided to try and get a degree in something I was interested in, and after getting the Fulbright scholarship I was ready (as in, my wallet) to take the challenge. Currently I am in my 3rd quarter of the Masters of Public Policy at the University of Chicago, where I participate actively in the following student organizations:
+
+- [Latin American Matters](http://www.lamuchicago.org/who-we-are/)
+
+- [University of Chicago Mexican Association](https://www.facebook.com/MexicansAtUChicago/)
+
+- [Chicago Policy Review](https://chicagopolicyreview.org/)
+
+For reference, here's a pic of me and my girlfriend at this year's mexican party:
+
+![Gf is the pretty one](https://raw.githubusercontent.com/NelsonPuc/myrepo/master/IMG_2838_1_20.jpeg)
+
```

Comment:
Merit: Fun picture! It is so joyful to read your biography, also with this nice pic!

Improvement: Probably I might have missed it, but i didn't see your reflections on the easy/hard part of this homework assignment. It would be nice to include a few sentences on this.